arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -1263,13 +1263,20 @@ IResearchAnalyzerFeature::IResearchAnalyzerFeature(application_features::Applica
   };
 }
 
+/*static*/ bool IResearchAnalyzerFeature::canUseVocbase(irs::string_ref const& vocbaseName,
```

Comment:
is this canUseVocbase  or canUse_Analyzer?

Reply:
it is  in general canUseAnalyzersFromVocbase but it seemed redundant as AnalyzerFeature implies analyzers so just canUseVocbase